python/sdk/prelude_sdk/controllers/partner_controller.py

```diff
@@ -33,10 +33,24 @@ def detach(self, partner_code: int):
         if res.status_code == 200:
             return res.json()
         raise Exception(res.text)
+
+    @verify_credentials
+    def block(self, partner_code: int, test_id: str):
+        """ Report to a partner to block a test """
+        params = dict(test_id=test_id)
```

Comment:
Why are we sending the test_id rather than the something unique to the activity like it's id? On the front end, per our convo yesterday it sounded like we wanted to send in a single activity rather than a test which would be a culmination of activities if they are ran multiple times. I could be mistaken though